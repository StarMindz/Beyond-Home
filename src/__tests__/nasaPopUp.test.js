import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import NasaPopUp from '../components/nasaPopUp';

describe('Homepage', () => {
  test('Show Correct render of NasaPopUp', () => {
    const popUp = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <NasaPopUp />
          </Router>
        </Provider>
      </React.StrictMode>,
    );
    expect(popUp).toMatchSnapshot();
  });
});
