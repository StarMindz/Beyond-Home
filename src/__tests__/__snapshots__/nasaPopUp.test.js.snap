// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage Show Correct render of NasaPopUp 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="popUp"
        id="0"
      >
        <h1
          class="popUpTitle"
        />
        <img
          alt="nasa"
          class="popUpImage"
          src=""
        />
        <p
          class="popUpDescription"
        />
        <p
          class="credit"
        >
          <span>
            Image Created by
          </span>
          <span />
          <span>
            on
          </span>
          <span />
        </p>
        <button
          class="nasaSecondButton"
          type="submit"
        >
          Back
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="popUp"
      id="0"
    >
      <h1
        class="popUpTitle"
      />
      <img
        alt="nasa"
        class="popUpImage"
        src=""
      />
      <p
        class="popUpDescription"
      />
      <p
        class="credit"
      >
        <span>
          Image Created by
        </span>
        <span />
        <span>
          on
        </span>
        <span />
      </p>
      <button
        class="nasaSecondButton"
        type="submit"
      >
        Back
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
